Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/moth.yml

```diff
@@ -0,0 +1,1210 @@
+        - ReagentId: Protein
+          Quantity: 4
+        - ReagentId: CapsaicinOil
+          Quantity: 1
+#Tastes like oats, sweet potato, carrot and parsnip
+
+- type: entity
+  name: mulan congee
+  parent: FoodBowlBase
+  id: FoodMothHuaMulanCongee
+  description: A smiley bowl of rice porridge with eggs and bacon.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - eggs
+      - bacon
+      - rice
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: huamulancongee
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 30
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 22
+        - ReagentId: Protein
+          Quantity: 8
+
+- type: entity
+  name: cornmeal porridge
+  parent: FoodBowlBase
+  id: FoodMothCornmealPorridge
+  description: A plate of cornmeal porridge. It's more flavourful than most porridges, and makes a good base for other flavours, too.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - corn
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: cornmealporridge
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 11
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 9
+        - ReagentId: Vitamin
+          Quantity: 2
+#Tastes like cornmeal.
+
+- type: entity
+  name: cheesy porridge
+  parent: FoodBowlBase
+  id: FoodMothCheesyPorridge
+  description: A rich and creamy bowl of cheesy cornmeal porridge.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - cheesy
+      - corn
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: cheesyporridge
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 20
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 12
+        - ReagentId: Vitamin
+          Quantity: 4
+        - ReagentId: Protein
+          Quantity: 4
+#Tastes like cornmeal, cheese, more cheese, lots of cheese.
+
+- type: entity
+  name: fried eggplant and polenta
+  parent: FoodBowlBase
+  id: FoodMothEggplantPolenta
+  description: Polenta loaded with cheese, served with a few discs of fried eggplant and some tomato sauce.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - corn
+      - cheesy
+      - eggplant
+      - tomato
+  - type: Sprite
+    sprite: Nyanotrasen/Objects/Consumable/Food/moth.rsi
+    state: eggplantpolenta
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxvol: 30
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Vitamin
+          Quantity: 6
+        - ReagentId: Protein
+          Quantity: 12
+#Tastes like cornmeal, cheese, eggplant and tomato sauce
+
+- type: entity
+  name: vegetarian chili
+  parent: FoodBowlBase
+  id: FoodMothVegetarianChili
+  description: For the hombres who don't want carne.
```

Comment:
"Vegan Chili. Made of real vegans."

Rosy's idea, and as always, I vibe with it.